editor/plugins/text_shader_editor.h

```diff
@@ -188,19 +189,21 @@ class TextShaderEditor : public MarginContainer {
 	void _bookmark_item_pressed(int p_idx);
 
 public:
+	void edit_shader(const Ref<Shader> &p_shader) override;
```

Comment:
```suggestion
	virtual void edit_shader(const Ref<Shader> &p_shader) override;
```
For all here, as per general style